tests/tracer/test_sampler.py

```diff
@@ -156,7 +179,7 @@ def test_sample_rate_deviation(self):
                     assert 0 == sample.get_metric(
                         SAMPLING_PRIORITY_KEY
                     ), "when priority sampling is on, priority should be 0 when trace is to be dropped"
-                assert_sampling_decision_tags(sample, agent=sample_rate)
+                assert_sampling_decision_tags(sample, agent=sample_rate, trace_tag="-1")
```

Comment:
comment here would be good, that is mean "dash one" and not "negative one", it took me a second to understand what this meant.

Reply:
or maybe what I actually want is clear doc string for the `assert_sampling_decision_tags` function ... ?